src/main/java/io/strimzi/test/container/StrimziKafkaContainer.java

```diff
@@ -278,7 +278,7 @@ protected void containerIsStarting(final InspectContainerResponse containerInfo,
             command += "bin/kafka-server-start.sh config/server.properties" + kafkaConfigurationOverride;
         } else {
             clusterId = this.randomUuid();
-            command += "bin/kafka-storage.sh format -t \"" + clusterId + "\" -c config/kraft/server.properties \n";
+            command += "bin/kafka-storage.sh format -t= \"" + clusterId + "\" -c config/kraft/server.properties \n";
```

Comment:
Should it be without the space?

```suggestion
            command += "bin/kafka-storage.sh format -t=\"" + clusterId + "\" -c config/kraft/server.properties \n";
```

Reply:
yes correct :) thanks